ports/stm32f4/common-hal/digitalio/DigitalInOut.c

```diff
@@ -43,11 +43,11 @@ digitalinout_result_t common_hal_digitalio_digitalinout_construct(
     self->pin = pin;
 
     GPIO_InitTypeDef GPIO_InitStruct = {0};
-    GPIO_InitStruct.Pin = (1 << pin->number);
+    GPIO_InitStruct.Pin = 1<<self->pin->number;
```

Comment:
Please swap the shifts back to pin_mask. Sorry I was confusing about the bitpacking stuff.

Reply:
Ok.